src/core/main.cpp

```diff
@@ -567,6 +567,72 @@ int main( int argc, char * * argv )
 		}
 	}
 
+	// Test file argument before continuing
+
+	QFile testFile( fileToLoad );
+	if( testFile.open( QIODevice::ReadOnly) )
+	{
+		// Test if render ( coreOnly ) and file is empty.
+		if( coreOnly == true && testFile.size() == 0 )
+		{
+			// is empty
+			printf( "The file %s is empty.\n",
+					fileToLoad.toStdString().c_str() );
+			exit( 1 );
+		}
+		testFile.close();
+	}
+
+	if( !fileToLoad.isEmpty() )
+	{
+		QFileInfo fileInfoLoad( fileToLoad );
+		if( !fileInfoLoad.exists() )
+		{
+			printf( "The file %s does not exist.\n",
+					fileToLoad.toStdString().c_str() );
+			exit( 1 );
+		}
+		else if( fileInfoLoad.isDir() )
+		{
+			printf( "%s is a directory.\n",
+					fileToLoad.toStdString().c_str() );
+			exit( 1 );
+		}
+		else if( !(	fileInfoLoad.suffix().toLower() == "mmp" ||
+				fileInfoLoad.suffix().toLower() == "mmpz" ||
+				fileInfoLoad.suffix().toLower() == "mpt" ||
+				fileInfoLoad.suffix().toLower() == "xml" ) )
```

Comment:
> It will open a blank project with the title you just gave it. I think at least we should not set the project title when the project couldn't be opened.

That's a regression from 1.1.3 which does this just fine...